rhel/SOURCES/grafana-5.3.2-change-icon.patch

```diff
@@ -12,15 +12,15 @@ index 5c51e28ef..b63edd2e6 100644
  
    </head>
 diff --git a/public/views/index.template.html b/public/views/index.template.html
-index 79da1d717..a672ab15b 100644
+index 096684d3e..286e2da1d 100644
 --- a/public/views/index.template.html
 +++ b/public/views/index.template.html
-@@ -13,7 +13,7 @@
+@@ -11,7 +11,7 @@
  
-   <link rel="stylesheet" href="public/build/grafana.[[ .Theme ]].css?v[[ .BuildVersion ]]">
+   <base href="[[.AppSubUrl]]/" />
  
 -  <link rel="icon" type="image/png" href="public/img/fav32.png">
 +  <link rel="icon" type="image/x-icon" href="public/img/percona-favicon.ico">
    <link rel="mask-icon" href="public/img/grafana_mask_icon.svg" color="#F05A28">
-   <link rel="apple-touch-icon" href="public/img/fav32.png">
- 
+   <link rel="apple-touch-icon" sizes="180x180" href="public/img/apple-touch-icon.png">
+   <meta name="apple-mobile-web-app-capable" content="yes">
```

Comment:
This file has to be removed. The same patch is already present in the PMM-2.0 branch
grafana-5.3.3-change-icon.patch